src/widgets/table.rs

```diff
@@ -194,6 +194,21 @@ impl HighlightSpacing {
     }
 }
 
+#[derive(Debug, Display, EnumString, PartialEq, Eq, Clone, Default, Hash)]
+pub enum HighlightArea {
```

Comment:
Please add a doc comment for the type (it's helpful when hovering over a type in front of a variable or field declaration).